src/Model.elm

```diff
@@ -56,6 +59,9 @@ type
     | Unreblog Int
     | Unreblogged (Result Mastodon.Error Mastodon.Status)
     | UserTimeline (Result Mastodon.Error (List Mastodon.Status))
+    | NewWebsocketUserMessage String
```

Comment:
Regarding the WebSocket renaming, I'm not sure. I want to be consistent everywhere, so `subscribeToWebSockets` is changed to `subscribeToWS` and `decodeWebSocketMessage` is changed to `decodeWSMessage`. Not sure if it's really readable.